crates/core/src/writer/record_batch.rs

```diff
@@ -65,7 +65,8 @@ impl RecordBatchWriter {
         partition_columns: Option<Vec<String>>,
         storage_options: Option<HashMap<String, String>>,
     ) -> Result<Self, DeltaTableError> {
-        let delta_table = DeltaTableBuilder::from_uri(table_uri)
+        let table_url = crate::table::builder::ensure_table_uri(table_uri)?;
```

Comment:
I will start complaining about fully qualified usage once it causes extra line breaks, let's see if that happens 😆